### PR TITLE
fix readme

### PR DESCRIPTION
**Pull request by ZhouXing19**
Note: the *Link* part is still TBD.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
 
-Details about and from the Cloud Report done in Q4 2020
+Details about and from the Cloud Report done in Q2 2022
 
-In 2018, we produced a "Cloud Report", wherein we assessed the performance of AWS vs. GCP along dimensions like TPC-C throughput, I/O, CPU and latency. This year, we aim to produce a similar report but including Azure.
+From 2018 to 2021, we produced the series of "Cloud Report", wherein we assessed the performance of AWS vs. GCP along dimensions like TPC-C throughput, I/O, CPU and latency.
 
 This repo will let us aggregate and share data among team members, including processes and results. 
 
+Resources for Cloud Report last year (2021)
```
Comment (srosenberg) [inline README.md:9]: Don't think we need this line; a bulleted list seems to be sufficient.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
 * [Highlights from the 2021 Cloud Report](https://www.cockroachlabs.com/blog/2021-cloud-report/)
 * [2021 Cloud Report](https://www.cockroachlabs.com/guides/2021-cloud-report)
```
Comment (srosenberg) [inline README.md:11]: Should we add the placeholders for the 2022 urls; I imagine they would be the same modulo `2022`.

**Comment by ZhouXing19** [comment]
Done changing the `Staff` part. Waiting for responses for the links.

**Review by srosenberg — APPROVED** [review]
Good to go after the new urls are added.

**Comment by ZhouXing19** [comment]
Thanks!

**Pull request merged**